backend/isic/crawlinginmyskin.py

```diff
@@ -0,0 +1,151 @@
+"""For further information about the API: https://isic-archive.com/api/v1/ """
+
+
+def ISIC_request(response, num=100):
+    """Fetching the metadata from images in ISIC archive by sending a HTTP
+    request to the ISIC's REST API.
+
+    Parameters:
+
+    num(int): Number of the images you get from the archive,default is 100
+
+    Returns:
+
+    Json: Response metadata in JSON format
+    """
+    start_url = f'https://isic-archive.com/api/v1/image?limit={num} \
+                  &sort=name&sortdir=-1&detail=true'
+    headers = {
+        'Accept': 'application/json'
+    }
+    r = requests.get(start_url, headers=headers)
+    data = r.json()
+    return data
+
+
+def check_dermoscopic(meta):
+    """Checking if a image is acquired through dermoscopy by ISIC's API.
+
+    Parameter:
+    meta: The metadata of the image getting through the API
+
+    Return:
+    True if the image is acquired through dermoscopy, False if it isn't
+    """
+
+    if "image_type" not in meta["meta"]["acquisition"]:
+        return False
+    elif not str(meta["meta"]["acquisition"]["image_type"]) == "dermoscopic":
+        return False
+    return True
+
+
+def check_melanoma(metadata):
+    """Checking if a image is confirmed melanocytic by ISIC's API.
+
+    Parameter:
+    metadata: The metadata of the image getting through the API
+
+    Return:
+    True if the image is confirmed melanocytic, False if it isn't
+    """
+    if "melanocytic" not in metadata["meta"]["clinical"]:
+        return False
+    elif not metadata["meta"]["clinical"]["melanocytic"]:
+        return False
+    return True
+
+
+def ISIC_getdata(numjson=200, numimage=20,
+                 dermo_filter=True,
+                 melano_filter=True,
+                 export=True):
+    """Download images from internet confirmed melanocytic in its metadata and
+    are classified as being acquired through dermoscopy and creating a CSV from
+    metadata of such images.
+
+    Parameters:
+    numjson(int): Number of metadata sets of images from ISIC_request().
+    Default is 100
+    numimage(int): Number of images you want fulfilling both requirements
+    (dermoscopic image and being melanocytic) from the metadata sets.
+    Should be more than numjson. Default is 20
+    dermo_filter(bool): Checking if you only want images acquired by dermoscopy
+    Default is true
+    melano_filter(bool): Checking if you only want images confirmed melanocytic
+    Default is True
+    export(bool): Option to choose if you want to export the metadata as a CSV
+    file. Default is True
+
+    Return:List of the metadata of the images downloaded
+    """
+    if (numjson < numimage or not
+            isinstance(numjson, int) or not
+            isinstance(numimage, int) or not
+            isinstance(dermo_filter, bool) or not
+            isinstance(melano_filter, bool) or not
+            isinstance(export, bool)):
+        print("Invalid parameters")
```

Comment:
You can `raise` an error here to escape the function and avoid the big `else`. However, since only I use it, you don't need to be this careful. Just drop this type checking.